lib/experimental/fonts-api/register-fonts-from-theme-json.php

```diff
@@ -59,8 +59,8 @@ function gutenberg_register_fonts_from_theme_json() {
 				$font_family['fontFace'] = (array) $font_family['fontFace'];
 
 				foreach ( $font_family['fontFace'] as $font_face ) {
-					// Skip if the webfont was registered through the Webfonts API.
-					if ( isset( $font_face['origin'] ) && 'gutenberg_wp_webfonts_api' === $font_face['origin'] ) {
+					// Skip if the font was registered through the Fonts API.
+					if ( isset( $font_face['origin'] ) && 'gutenberg_wp_fonts_api' === $font_face['origin'] ) {
```

Comment:
Thinking a little more about this ...

User selected fonts are stored in the database, i.e. in the `'wp_global_styles'` custom post type. This means the origin could already exist in the database. Thus, this won't work for saved global style typography.

Instead, will need to check for either `'gutenberg_wp_fonts_api'` OR `'gutenberg_wp_webfonts_api'`.

```suggestion
					if ( 
						isset( $font_face['origin'] ) && 	
						(
							'gutenberg_wp_fonts_api' === $font_face['origin'] ||
							'gutenberg_wp_webfonts_api' === $font_face['origin']
						)
					) {
```

Reply:
Never mind. On further testing, `'origin'` is not saved to the database. That's great news as it eliminates the need for a migration plan when this API is introduced into WordPress Core 😅